docs/ingest.md

```diff
@@ -0,0 +1,44 @@
+# Ingesting Assets
+
+Every [asset](index.md#assets) belongs to a project. 
+Assets are group by Projects, you should create one using the [Project Administration](https://docs.dasch.swiss/latest/DSP-APP/user-guide/project/?h=project).
```

Comment:
```suggestion

```

I don't think this second sentence fits there very well: Firstly, everyone doing a bulk ingest knows how to create a project; secondly, one can just as well use DSP-TOOLS to create projects; thirdly "using the Project Administration" sounds strange to my ears.  
In the interest of a documentation that's short and to the point, I would simply delete this sentence.